internal/quickstart/messages.go

```diff
@@ -60,6 +60,35 @@ func confirmedFlag(flag flag) tea.Cmd {
 	}
 }
 
+type msgRequestError struct {
```

Comment:
Creates a new type to encapsulate converting from the API response to something we handle in messages and models.